theatre/studio/src/StudioStore/StudioStore.ts

```diff
@@ -54,13 +55,16 @@ export default class StudioStore {
   private readonly _atom: Atom<FullStudioState>
   readonly atomP: Pointer<FullStudioState>
 
+  private _copiedKeyframes: Keyframe[]
```

Comment:
One downside is that this doesn't get persisted.

Reply:
To persist, this is where the persisted state should go: https://github.com/AriaMinaei/theatre/blob/cf3858d121925261ac5203a30833daaf1d9ad460/theatre/studio/src/store/types/ahistoric.ts#L4

Reply:
And here is an example of a state editor fn: https://github.com/AriaMinaei/theatre/blob/cf3858d121925261ac5203a30833daaf1d9ad460/theatre/studio/src/store/stateEditors.ts#L317